doc/ref/gappkg.xml

```diff
@@ -236,8 +236,9 @@ we provide guidelines for the release preparation and its distribution.
 <Heading>Structure of a GAP Package</Heading>
 
 <Index Subkey="for a GAP package">home directory</Index>
-A &GAP; package should have an alphanumeric name (<A>packagename</A>, say);
-mixed case is fine, but there should  be  no  whitespace characters. 
+A &GAP; package should have an alphanumeric name (let us say
+<A>packagename</A>);
```

Comment:
Here I would simple remove the complete text between `(` and `)`, as it is irrelevant for that sentence; in fact I think it has a negative contribution in that it might confuse (it could be read as being an example for an alphanumeric name, which it of course is; but it really is meant to be the placeholder name for a hypothetical package the rest of the text talks about). 

I'd just remove it, as the next sentence "introduces" it sufficiently on its own, IMHO.

Reply:
Agreed.